### ISSUE TITLE
Enable FPPMon on Docker

### ISSUE THREAD

**Issue by kevinsaucier** [KulpLights/fpp-FPPMon#11]
I expect you already know, but FPPMon doesn't appear to work on FPP in docker.  It just says it's not running and to restart FPPD to start it.  No luck.

**Comment by kevinsaucier** [comment]
So, I just noticed that, while the FPPMon page doesn't show the username/password fields in the regular UI (see pic), when the page fails to render properly and just shows the 'text' version of the code, the name/password boxes were there.  Of course, I have no idea how to make it fail to render (it just happens every so often and a refresh of the pages fixes it) and I forgot to grab a screenshot before refreshing the page.  :(   Maybe it's just a matter of the boxes not showing up (as in the Home Assistant Plugin bug that Daniel fixed a few weeks ago).  🤞

![image](https://github.com/KulpLights/fpp-FPPMon/assets/25715814/45b216c0-8397-4013-a72d-0fab8d90fa9f)

**Comment by dkulp** [comment]
At this point, the FPPMon plugin requires either a Raspberry Pi or Beaglebone.   Docker and Mac are currently unsupported.

**Comment by kevinsaucier** [comment]
> At this point, the FPPMon plugin requires either a Raspberry Pi or Beaglebone. Docker and Mac are currently unsupported.

That's a bummer.  Appreciate the response, though.  👍 

I really enjoyed the alternative to a squished webpage during Halloween and was hoping to use it for Xmas as well, assuming I stick with Docker for Xmas.

**Comment by dkulp** [comment]
The plugin itself does not have to run on one of the monitored FPP instances.  If you have FPP running on a Pi or BBB, you can install it there and still monitor the Docker instance.

**Comment by kevinsaucier** [comment]
Good point.  I'm planning to leave my existing dedicated BBG running (for now, at least) as a backup when I start messing with Docker anyway so I'll just use that as the jump point.  Again, really appreciate your time (and your great controllers/code)!
